works.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="work-styles.css">
+</head>
+<body>
+
+  <header>
+    <h1>My Recent Works</h1>
+  </header>
+<hr/>
+  <section class="featured">
+    <img class="gymfit" src="./images/Featured work.png" alt="Featured Work">
+    <h2 class="post-stories">Multi-Post Stories</h2>
+    <p class="feat-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <div class="lang-4">
+    <p class="css">css</p>
+    <p class="html">html</p>
+    <p class="bootstrap">bootstrap</p>
+    <p class="Ruby">Ruby</p>
```

Comment:
- [ ] For semantic reasons, Kindly make sure the list of project technologies (e.g., `JavasScript`, `HTML`, and `CSS`) are a list `ul`, and each of them a `li` and not a generic paragraph since it's a list of elements